src/components/organisms/WithInformationCard/WithInformationCard.tsx

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+
+import { AnyVenue } from "types/venues";
+
+import { InformationLeftColumn } from "components/organisms/InformationLeftColumn";
+import InformationCard from "components/molecules/InformationCard";
+
+import "./WithInformationCard.scss";
+
+interface PropsType {
+  venue: AnyVenue;
+  showInfo?: boolean;
+}
+
+export const WithInformationCard: React.FunctionComponent<PropsType> = ({
+  venue,
+  showInfo = false,
+}) => (
+  <>
+    {showInfo && (
```

Comment:
Why do we pass this prop in to conditionally render here, rather than just conditionally rendering `WithInformationCard` itself in the calling code/parent component?

```suggestion
}) => (
```